homeassistant/components/renson/sensor.py

```diff
@@ -0,0 +1,310 @@
+    ),
+    RensonSensorEntityDescription(
+        key="CURRENT_LEVEL_FIELD",
+        name="Ventilation level",
+        state_class=None,
+        field=CURRENT_LEVEL_FIELD,
+        raw_format=True,
+    ),
+    RensonSensorEntityDescription(
+        key="CURRENT_AIRFLOW_EXTRACT_FIELD",
+        name="Total airflow out",
+        field=CURRENT_AIRFLOW_EXTRACT_FIELD,
+        raw_format=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="m³/h",
+    ),
+    RensonSensorEntityDescription(
+        key="CURRENT_AIRFLOW_INGOING_FIELD",
+        name="Total airflow in",
+        field=CURRENT_AIRFLOW_INGOING_FIELD,
+        raw_format=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="m³/h",
+    ),
+    RensonSensorEntityDescription(
+        key="OUTDOOR_TEMP_FIELD",
+        name="Outdoor air temperature",
+        field=OUTDOOR_TEMP_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    RensonSensorEntityDescription(
+        key="INDOOR_TEMP_FIELD",
+        name="Extract air temperature",
+        field=INDOOR_TEMP_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    RensonSensorEntityDescription(
+        key="FILTER_REMAIN_FIELD",
+        name="Filter change",
+        field=FILTER_REMAIN_FIELD,
+        raw_format=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="days",
+    ),
+    RensonSensorEntityDescription(
+        key="HUMIDITY_FIELD",
+        name="Relative humidity",
+        field=HUMIDITY_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="%",
+    ),
+    RensonSensorEntityDescription(
+        key="MANUAL_LEVEL_FIELD",
+        name="Manual level",
+        field=MANUAL_LEVEL_FIELD,
+        raw_format=False,
+        state_class=None,
+    ),
+    RensonSensorEntityDescription(
+        key="BREEZE_TEMPERATURE_FIELD",
+        name="Breeze temperature",
+        field=BREEZE_TEMPERATURE_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="BREEZE_LEVEL_FIELD",
+        name="Breeze level",
+        field=BREEZE_LEVEL_FIELD,
+        raw_format=False,
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="DAYTIME_FIELD",
+        name="Start day time",
+        field=DAYTIME_FIELD,
```

Comment:
What kind of value is this?

Reply:
This is a hour field HH:MM (24h). What class can I use for this?

Reply:
We don't have a device class for just time at the moment.